tendrl/gluster_integration/objects/volume/atoms/expand/__init__.py

```diff
@@ -0,0 +1,89 @@
+import subprocess
+
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons import objects
+from tendrl.gluster_integration.objects.volume import Volume
+
+
+class Expand(objects.BaseAtom):
+    obj = Volume
```

Comment:
This is not required since  a long time now, atoms are registered to objects under whose directory they are defined